### ISSUE TITLE
Typing mismatch: dist

### ISSUE THREAD

**Issue by ukch** [outline/rich-markdown-editor#254]
When using TypeScript, it is not possible to import objects unless they are one of the module exports from the main module (this happens with e.g. `Mark`), because of missing type definitions. The definitions do exist, but they are linked to the original source files, not the files in dist.

**To Reproduce**
1. `import Mark from "rich-markdown-editor/dist/marks/Mark";` (e.g. in order to subclass it to make an extension)

**Expected behavior**
TypeScript should use the type definition from `rich-markdown-editor/dist/@types/marks/Mark`

**Actual behaviour**
TypeScript cannot find the definition and throws an error. A workaround is to add the following definition to `react-app-env.d.ts`:

```typescript
declare module 'rich-markdown-editor/dist/marks/Mark' {
    import Mark from "rich-markdown-editor/dist/@types/marks/Mark";
    export default Mark;
}
```

**Version**
10.6.2

**Comment by tommoor** [comment]
I'm not familiar enough with typescript to know what's going on here, seemingly all the pieces are in place for the types to be found. Perhaps it's the extra level of nesting of `@types` within `/dist` ?

**Comment by tommoor** [comment]
Added a possible fix in `v10.6.4` colocated the .ts files, let me know if that works…

**Comment by ukch** [comment]
Thanks @tommoor. That seems to have done the trick!
